configuration/configuration.go

```diff
@@ -646,6 +635,40 @@ type Proxy struct {
 	Password string `yaml:"password"`
 }
 
+type Validation struct {
+	// Enabled enables the other options in this section. This field is
+	// deprecated in favor of Disabled.
+	Enabled bool `yaml:"enabled,omitempty"`
+	// Disabled disables the other options in this section.
+	Disabled bool `yaml:"disabled,omitempty"`
+	// Manifests configures manifest validation.
+	Manifests ValidationManifests `yaml:"manifests,omitempty"`
+}
+
+type ValidationManifests struct {
+	// URLs configures validation for URLs in pushed manifests.
+	URLs struct {
+		// Allow specifies regular expressions (https://godoc.org/regexp/syntax)
+		// that URLs in pushed manifests must match.
+		Allow []string `yaml:"allow,omitempty"`
+		// Deny specifies regular expressions (https://godoc.org/regexp/syntax)
+		// that URLs in pushed manifests must not match.
+		Deny []string `yaml:"deny,omitempty"`
+	} `yaml:"urls,omitempty"`
+	// ImageIndex configures validation of image indexes
+	ImageIndex ValidationManifestsImageIndex `yaml:"imageindex,omitempty"`
+}
+
+type ValidationManifestsImageIndex struct {
+	Images struct {
+		// Exist validates that images referenced by indexes exist in the registry
+		// before the image index is accepted.
+		Exist *bool `yaml:"exist"`
```

Comment:
The implication here is that this would default to false, and thus registries would default to not requiring the referenced images to exist?

is that the default we actually want? (it seems misaligned to current registry behavior which, as far as i know, typically enforces that the images must exist)

Reply:
also not sure if there's value in making it a pointer.  Are you trying to preserve the distinction between "defaulted false" and "user set it to false"?

Reply:
It's actually default true where is used, and yes, that's why it's a pointer, so you can distinguish unspecified from specified and use the right default.

Reply:
defaulting to true feels unusual to me in my experience w/ apis and CLI flags, so i'd consider renaming the field to something where default false gets you the behavior you want.  But if you would prefer not to do that, the definitely add some godoc that makes it clear that the behavior defaults to true if unspecified.